tests/model_registry/rest_api/conftest.py

```
@@ -63,3 +92,250 @@ def updated_model_registry_resource(
+    """
+    CA_CONFIGMAP_NAME = request.param.get("ca_configmap_name", "mysql-ca-configmap")
+    CA_MOUNT_PATH = request.param.get("ca_mount_path", "/etc/mysql/ssl")
+    deployment = Deployment(
+        client=admin_client,
+        name=model_registry_db_deployment.name,
+        namespace=model_registry_namespace,
+    )
+    deployment.wait_for_condition(condition="Available", status="True")
+    original_deployment = deployment.instance.to_dict()
+    spec = original_deployment["spec"]["template"]["spec"]
+    my_sql_container = next(container for container in spec["containers"] if container["name"] == "mysql")
```

Pylint (3.3.7)

[refactor] Do not raise StopIteration in generator, use return statement instead

(R1708)